code_samples/project_view_pane/README.md

```diff
@@ -0,0 +1,25 @@
+# Project View Pane Demo [![JetBrains IntelliJ Platform SDK Docs](https://jb.gg/badges/docs.svg)][docs]
+*Reference: [Plugin Services in IntelliJ SDK Docs][docs:project_view]*
+
+## Quickstart
+
+The current demo describes an implementation of the `projectViewPane` extension point, which allows to create
+an additional presentation type for the Project view pane. `ImagesProjectViewPane` limits the project tree to the images
+only.
```

Comment:
Suggest:_The Project View Pane plugin has an implementation of the `projectViewPane` extension point, which supports creating new presentation types for the Project view pane. The `ImagesProjectViewPane` implementation in this code sample limits the project tree to only images._